### ISSUE TITLE
Memory leak when using Stress Type Visualization mod

### ISSUE THREAD

**Issue by Amaroq-Clearwater** [56kyle/StressTypeVisualization#2]
Having this mod enabled causes a memory leak in the game, severely degrading performance.

**Comment by 56kyle** [comment]
yeah I definitely noticed a decrease in performance while using it, although at the time I attributed it to having to recreate each pieces material. Will look back into it soon
